README.md

```diff
@@ -8,6 +8,7 @@ Name                                                 | Style                 | N
 -----------------------------------------------------|-----------------------|------
 No-op                                                | N/A                   | N/A
 [argh](https://github.com/google/argh)               | `derive`              |
+[bpaf](https://github.com/pacak/bpaf)                | Imperative            |
```

Comment:
So the question is what should be our bar for including a parser?

It seems there has been a flood of argparsers recently and it'd be cumbersome to include them all and hard to keep up to date with their passing support status.  Based on popularity, currently the least popular is 45k recent downloads.  Unsure what other metrics and criteria we should use.  It'd be nice to capture and raise visibility of "up and coming" ones (iirc lexopt originally fit in that category though I don't remember how popular it was by the time we included it).

Reply:
I guess you have to answer this question. :) I liked the library because it tries to fill it's only niche, having a fast compile time but neglecting the run time. And, at least to me the simple imperative syntax is much easier on the eyes than, say, clap.

Reply:
Personally, I find the combinator style intriguing but annoying because it reads bottom up instead of top-down.

However, I think I'm going to go ahead with this because the combinator approach is unique.  I'll also update the descriptions to be more specific.

Reply:
> I find the combinator style intriguing but annoying because it reads bottom up instead of top-down.

Now you can write it top-down too: https://github.com/pacak/bpaf/blob/master/examples/top_to_bottom.rs